src/ui/views/Calendar/calendar.ts

```diff
@@ -3,6 +3,8 @@ import { get } from "svelte/store";
 
 import { isDate, type DataRecord } from "src/lib/dataframe/dataframe";
 import { i18n } from "src/lib/stores/i18n";
+import type { FirstDayOfWeek } from "src/settings/settings";
+import moment from "moment";
```

Comment:
Can this be imported directly from "obsidian"?

Reply:
You mean `moment`? I believe it is? The package.json doesn't list moment as a dependency.

Hm. Looking at the esbuild config, it's not listed as external. Maybe it should be.

Reply:
I saw this guidelin from new plugin review. e.g. https://github.com/obsidianmd/obsidian-releases/pull/3501#issuecomment-2112816825

However the namespace import style prevents me from adapting `import moment from "moment"` to `import { moment } from "obsidian"`

https://github.com/obsidianmd/obsidian-api/blob/8b2eda0f24285636c8aa116972643e5233a23dc1/obsidian.d.ts#L8C40-L9C1

Umm seem like we have to keep the original syntax. Thanks for the reply and as there's no more action on my end, I'm marking this as resolved.